lnwallet/reservation.go

```diff
@@ -58,23 +58,23 @@ type InputScript struct {
 }
 
 // ChannelReservation represents an intent to open a lightning payment channel
-// a counterparty. The funding processes from reservation to channel opening is
-// a 3-step process. In order to allow for full concurrency during the
+// with a counterparty. The funding processes from reservation to channel opening
```

Comment:
Could you make the changes not in `fundingmanager` a separate commit labeled `lnwallet:`? :)

Reply:
Done